platform-dom/src/main/java/com/softicar/platform/dom/elements/number/decimal/DomDecimalDisplay.java

```diff
@@ -0,0 +1,54 @@
+package com.softicar.platform.dom.elements.number.decimal;
+
+import com.softicar.platform.common.core.locale.CurrentLocale;
+import com.softicar.platform.common.core.number.BigDecimalMapper;
+import com.softicar.platform.common.core.number.formatter.BigDecimalFormatter;
+import com.softicar.platform.common.core.number.formatter.BigDecimalScaleApplier;
+import com.softicar.platform.dom.element.IDomElement;
+import com.softicar.platform.dom.elements.DomDiv;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+/**
+ * An {@link IDomElement} to display decimal values according to
+ * {@link CurrentLocale}.
+ *
+ * @author Oliver Richers
+ */
+public class DomDecimalDisplay<T extends Number> extends DomDiv {
+
+	private final BigDecimalMapper<T> mapper;
+	private Integer scale;
+
+	public DomDecimalDisplay(BigDecimalMapper<T> mapper) {
+
+		this.mapper = mapper;
```

Comment:
```suggestion
		this.mapper = Objects.requireNonNull(mapper);
```
Let's fail early here.

Reply:
did so